pytradfri/device/__init__.py

```diff
@@ -121,48 +136,48 @@ class DeviceInfo:
     }
     ATTR_BATTERY = "9"
 
-    def __init__(self, device):
+    def __init__(self, device: Device) -> None:
         """Create object of class."""
         self._device = device
 
     @property
-    def manufacturer(self):
+    def manufacturer(self) -> str:
         """Human readable manufacturer name."""
-        return self.raw.get(DeviceInfo.ATTR_MANUFACTURER)
+        return str(self.raw.get(DeviceInfo.ATTR_MANUFACTURER))
```

Comment:
We don't want to convert `None` to a string.

Reply:
Corrected.